src/Microsoft.ML.Transforms/BootstrapSampleTransform.cs

```diff
@@ -84,10 +84,11 @@ public BootstrapSampleTransform(IHostEnvironment env, Arguments args, IDataView
         }
 
         /// <summary>
```

Comment:
> /// <summary> [](start = 7, length = 14)

just double-checking the expectations: this work is for the new api, right? Those would not translate to the constructors of the CSharpAPI

Reply:
Yes, this is only for core components and only for the convenience constructors that I added recently. I have referenced the issue #524 in the description which will tell you which of the core transforms will be targeted in this work.

Reply:
Yes, this is only for core components and only for the convenience constructors that I added recently. I have referenced the issue #524 in the description which will tell you which of the core transforms will be targeted in this work.

---
In reply to: [207689062](https://github.com/dotnet/machinelearning/pull/646#discussion_r207689062) [](ancestors = 207689062)